README.md

```diff
@@ -34,175 +34,3 @@
 | ✨ | **Reactive.** (Optional) [RxJS](https://github.com/ReactiveX/rxjs) API |
 | 🔗 | **Relational.** Built on rock-solid [SQLite](https://www.sqlite.org) foundation |
 | ⚠️ | **Static typing** with [Flow](https://flow.org) or [TypeScript](https://typescriptlang.org) |
-
```

Comment:
hmm, I'd like to keep README the way it was before - it's used on the GitHub page, not just on docs website

Reply:
hmm, I'm not sure yet. I worry this would be an extra maintenance burden to have to update both the main README and the docs

Reply:
@radex Sorry, just checked and realized that this is already in a separate file from the one used by the main README so I think it should be ok?

Reply:
ok I'll revert this change! Maybe someone can figure out a script to copy based off the sections.  I tried asking around some forums and googling but to no avail ahha